Set `repository.url` in generated npm packages for deltachat-rpc-server

Currently automated publishing fails with this error:
```
2024-05-15T22:41:45.7862161Z npm notice 
2024-05-15T22:41:45.7865836Z npm notice 📦  @deltachat/stdio-rpc-server@1.138.4
2024-05-15T22:41:45.7866736Z npm notice === Tarball Contents === 
2024-05-15T22:41:45.7887083Z npm notice 3.5kB README.md    
2024-05-15T22:41:45.7888173Z npm notice 987B  index.d.ts   
2024-05-15T22:41:45.7888747Z npm notice 4.8kB index.js     
2024-05-15T22:41:45.7891150Z npm notice 994B  package.json 
2024-05-15T22:41:45.7891963Z npm notice 182B  src/const.js 
2024-05-15T22:41:45.7892723Z npm notice 1.5kB src/errors.js
2024-05-15T22:41:45.7893502Z npm notice === Tarball Details === 
2024-05-15T22:41:45.7901303Z npm notice name:          @deltachat/stdio-rpc-server             
2024-05-15T22:41:45.7902703Z npm notice version:       1.138.4                                 
2024-05-15T22:41:45.7904232Z npm notice filename:      deltachat-stdio-rpc-server-1.138.4.tgz  
2024-05-15T22:41:45.7905700Z npm notice package size:  4.4 kB                                  
2024-05-15T22:41:45.7906792Z npm notice unpacked size: 12.0 kB                                 
2024-05-15T22:41:45.7907899Z npm notice shasum:        eb65cffd2439f23601b003f0be2d8160553b9228
2024-05-15T22:41:45.7909130Z npm notice integrity:     sha512-atXC0JkvWBHCv[...]L75KdVPaGVKCg==
2024-05-15T22:41:45.7910210Z npm notice total files:   6                                       
2024-05-15T22:41:45.7910953Z npm notice 
2024-05-15T22:41:45.7925690Z npm notice Publishing to https://registry.npmjs.org/ with tag latest and default access
2024-05-15T22:41:46.5457615Z npm notice publish Signed provenance statement with source and build information from GitHub Actions
2024-05-15T22:41:46.5459695Z npm notice publish Provenance statement published to transparency log: https://search.sigstore.dev/?logIndex=94181018
2024-05-15T22:41:47.1128612Z npm ERR! code E422
2024-05-15T22:41:47.1140874Z npm ERR! 422 Unprocessable Entity - PUT https://registry.npmjs.org/@deltachat%2fstdio-rpc-server - Error verifying sigstore provenance bundle: Failed to validate repository information: package.json: "repository.url" is "", expected to match "https://github.com/deltachat/deltachat-core-rust" from provenance
2024-05-15T22:41:47.1148004Z 
2024-05-15T22:41:47.1149391Z npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/2024-05-15T22_41_45_371Z-debug-0.log
2024-05-15T22:41:47.1219385Z ##[error]Process completed with exit code 1.
```

This happens when attempting to run this line:
https://github.com/deltachat/deltachat-core-rust/blob/4a738ebd195a426b114558d82173bdbc0e2f13cc/.github/workflows/deltachat-rpc-server.yml#L404

Link to CI run:
https://github.com/deltachat/deltachat-core-rust/actions/runs/9103223092/job/25025780633

--- a/deltachat-rpc-server/npm-package/package.json
+++ b/deltachat-rpc-server/npm-package/package.json
@@ -1,6 +1,10 @@
 {
   "license": "MPL-2.0",
   "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/deltachat/deltachat-core-rust.git"
+  },
   "name": "@deltachat/stdio-rpc-server",
   "optionalDependencies": {},
   "peerDependencies": {

--- a/deltachat-rpc-server/npm-package/scripts/src/make_package.py
+++ b/deltachat-rpc-server/npm-package/scripts/src/make_package.py
@@ -13,14 +13,21 @@ def write_package_json(platform_path, rust_target, my_binary_name):
     tomlfile = open("../../Cargo.toml", 'rb')
     version = tomllib.load(tomlfile)['package']['version']
 
-    package_json = dict({
-        "name": "@deltachat/stdio-rpc-server-" + convert_os_to_npm_os(os) + "-" + convert_cpu_arch_to_npm_cpu_arch(cpu_arch),
+    package_json = {
+        "name": "@deltachat/stdio-rpc-server-"
+        + convert_os_to_npm_os(os)
+        + "-"
+        + convert_cpu_arch_to_npm_cpu_arch(cpu_arch),
         "version": version,
         "os": [convert_os_to_npm_os(os)],
         "cpu": [convert_cpu_arch_to_npm_cpu_arch(cpu_arch)],
         "main": my_binary_name,
-        "license": "MPL-2.0"
-    })
+        "license": "MPL-2.0",
+        "repository": {
+            "type": "git",
+            "url": "https://github.com/deltachat/deltachat-core-rust.git",
+        },
+    }
 
     file = open(platform_path + "/package.json", 'w')
     file.write(json.dumps(package_json, indent=4))

